standalone/README.md

```diff
@@ -1,4 +1,6 @@
 Standalone scripts
 ==================
 
-Scripts that are self contained and are only run when your start them manually. 
\ No newline at end of file
+Scripts that are self contained and are only run when your start them manually. 
```

Comment:
Sure, now I just need to figure out how to amend a PR submitted through the web interface :)